src/main/java/com/shopping_app/shoppingApp/model/User/Request/UserRegisterRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.shopping_app.shoppingApp.model.User.Request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.shopping_app.shoppingApp.domain.Address;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotBlank;
+import java.util.Set;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UserRegisterRequest {
+
+    @NotBlank
+    private String name;
+
+    @NotBlank
+    private String email;
+
+    @NotBlank
+    private String phoneNumber;
+
+    @NotBlank
+    private String password;
+    
+    private Set<Address> address;
```

Comment:
Instead of using `Address` domain directly, use request class.

Reply:
done